### PR TITLE
Adding optional support for trailing slash clean in PATH

### PR DESCRIPTION
**Pull request by rippak**
Moved one old project to F3 where trailing slashes in routes are obligatory

### CODE REVIEW

File: base.php [hunk 1]
```diff
@@ -2530,6 +2531,7 @@ function($level,$text,$file,$line) {
 			'PACKAGE'=>self::PACKAGE,
 			'PARAMS'=>[],
 			'PATH'=>$path,
+            'PATH_SLASH_CLEAN'=>TRUE,
```
Comment (Rayne) [inline base.php:2534]: When this gets accepted, make all indentations consistent and don't forget to update the documentation. :-)

Comment (rippak) [reply base.php:2534]: Thanks, I'll check it. Phpstorm shows normal indents. 
Where should I update the docs?

Comment (Rayne) [reply base.php]: The docs are [here](https://github.com/f3-factory/F3com-data), but I would wait until this change got accepted.

Comment (rippak) [reply base.php]: Of course, just asking

**Comment by ikkez** [comment]
Would you mind if we rename the option var to something like... `REROUTE_TRAILING_SLASH` maybe? 🤔 
I'm wondering if there's more to do here. Like generating links to aliases? Switching to trailing slashes and also reroute non-trailing slash urls to those with slash, when the new option is on. Having both options enabled might lead to duplicate content warning .. just keep that in mind.

**Comment by rippak** [comment]
Naming is not a problem at all.

**Comment by ikkez** [comment]
Renamed to `REROUTE_TRAILING_SLASH` and committed.

**Pull request merged**